go/apps/http_api_nostream/vumi_app.py

```diff
@@ -326,7 +326,12 @@ def schedule_push_retry(self, user_account_key, url, method, data,
             resp = yield http_request_full(
                 retry_url, data=retry_data, headers=retry_headers,
                 timeout=self.http_retry_timeout)
-            if not (200 <= resp.code < 300):
+            if resp.code == 429:
+                log.warning(
+                    "Retrying is rate limited for account '%s'."
+                    " Discarding HTTP request." % (user_account_key,))
+                returnValue(None)
```

Comment:
I think I might be reading this wrong, but do we maybe need a `return` below this `returnValue` to avoid us reaching the `Succesfully scheduled...` logging at the bottom of the method?